File: 1945. Sum of Digits of String After Convert.java
class Solution {
    public int getLucky(String s, int k) {
        int answer = 0;
        for (char c : s.toCharArray()) answer += getDigitSum(c-'a'+1);
        while (k-- > 1) answer = getDigitSum(answer);
        return answer;
    }
    private int getDigitSum(int i) {
        int answer = 0;
        while (i > 0) {
            answer += i % 10;
            i /= 10;
        }
        return answer;
    }
}

File: 1945. Sum of Digits of String After Convert.go
func getLucky(s string, k int) int {
    answer := 0
    for _, v := range s {
        answer += getDigitSum(int(v-'a'+1))
    }
    for k > 1 {
        answer = getDigitSum(answer)
        k--
    }
    return answer
}

func getDigitSum(i int) int {
    answer := 0
    for i > 0 {
        answer += i % 10
        i /= 10
    }
    return answer
}

